src/zeroD/ReactorNet.cpp

```diff
@@ -256,7 +257,10 @@ void ReactorNet::eval(doublereal t, doublereal* y,
     m_time = t; // This will be replaced at the end of the timestep
     updateState(y);
     for (size_t n = 0; n < m_reactors.size(); n++) {
-        m_reactors[n]->evalEqs(t, y + m_start[n], ydot + m_start[n], p);
+        // Need to implement DAE solver to use evalEqs, due to volume component
```

Comment:
Most importantly, the `DenseMatrix` implementation still results in a singular Jacobian and cannot be solved with the existing `evalEqs()` function... this is because the dV/dt calculated here does not directly depend on reactor volume. Need to look into adding DAE solution capability, or modifying the volume equation in `evalEqs()`.

Reply:
Resolved by paulblum/cantera@8bd7385